File: app/src/main/java/org/com/castcodechallenge/comcastcodechallenge/api/model/RelatedTopic.java

package org.com.castcodechallenge.comcastcodechallenge.api.model;

import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;

public class RelatedTopic {

    @SerializedName("FirstURL")
    @Expose
    private String firstURL;
    @SerializedName("Icon")
    @Expose
    private Icon icon;
    @SerializedName("Result")
    @Expose
    private String result;
    @SerializedName("Text")
    @Expose
    private String text;

    public String getFirstURL() {
        return firstURL;
    }

    public void setFirstURL(String firstURL) {
        this.firstURL = firstURL;
    }

    public Icon getIcon() {
        return icon;
    }

    public void setIcon(Icon icon) {
        this.icon = icon;
    }

    public String getResult() {
        return result;
    }

    public void setResult(String result) {
        this.result = result;
    }

    public String getText() {
        return text;
    }

    public void setText(String text) {
        this.text = text;
    }

}

File: app/src/main/java/org/com/castcodechallenge/comcastcodechallenge/api/model/RelatedTopic.kt
package org.com.castcodechallenge.comcastcodechallenge.api.model

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class RelatedTopic {

    @SerializedName("FirstURL")
    @Expose
    var firstURL: String? = null
    @SerializedName("Icon")
    @Expose
    var icon: Icon? = null
    @SerializedName("Result")
    @Expose
    var result: String? = null
    @SerializedName("Text")
    @Expose
    var text: String? = null

}
